responses.py

```diff
@@ -25,6 +25,11 @@
     from urllib import quote
 else:
     from urllib.parse import urlparse, parse_qsl, urlsplit, urlunsplit, quote
+    
+if sys.version_info[:2] < (3,4):
+    from cookies import Cookies
+else:
+    import biscuits
```

Comment:
Couldn't this also be 

```python
try:
    import biscuits
    use_biscuits = True
except ImportError:
    use_biscuits = False
    from cookies import Cookies
```

That would let you reduce the number of places you check the python runtime version, as you could use `if use_biscuits` instead.

An alternative approach would be for responses to provide an abstraction around the two libraries and handle the differences there.

Reply:
Yes, I like the abstraction approach. I'll make an update.